config/filter.d/sshd.conf

```diff
@@ -31,11 +29,11 @@ failregex = ^%(__prefix_line)s(?:error: PAM: )?[aA]uthentication (?:failure|erro
             ^%(__prefix_line)sUser .+ from <HOST> not allowed because a group is listed in DenyGroups\s*$
             ^%(__prefix_line)sUser .+ from <HOST> not allowed because none of user's groups are listed in AllowGroups\s*$
             ^(?P<__prefix>%(__prefix_line)s)User .+ not allowed because account is locked<SKIPLINES>(?P=__prefix)(?:error: )?Received disconnect from <HOST>: 11: .+ \[preauth\]$
-            ^(?P<__prefix>%(__prefix_line)s)Disconnecting: Too many authentication failures for .+? \[preauth\]<SKIPLINES>(?P=__prefix)(?:error: )?Connection closed by <HOST> \[preauth\]$
```

Comment:
Still again, it would be too aggressive for normal fail2ban usage. I've made another PR (#1209, but is a little bit out-dated) where it can be extended possibly, but even there I'm not sure that would be good thing.
SSHD is even more global service as dovecot, to do something rash just so (it is a base for many-many other tasks, begins with git and ends with sftp...)